packages/material-ui-styled-engine-sc/src/index.js

```diff
@@ -1,14 +1,27 @@
 import scStyled from 'styled-components';
 
 export default function styled(tag, options) {
+  let stylesFactory;
+
   if (options) {
-    return scStyled(tag).withConfig({
+    stylesFactory = scStyled(tag).withConfig({
       displayName: options.label,
       shouldForwardProp: options.shouldForwardProp,
     });
   }
 
-  return scStyled(tag);
+  stylesFactory = scStyled(tag);
+
+  if (process.env.NODE_ENV !== 'production') {
+    return (...styles) => {
+      if (styles.some((style) => style === undefined)) {
+        console.error('empty', options.label);
```

Comment:
A test that explicitly matches the error message with `toErrorDev` would be helpful here. It's not clear that this error message is helpful.